manticore/native/cpu/abstractcpu.py

```diff
@@ -527,9 +527,10 @@ def __init__(self, regfile: RegisterFile, memory: Memory, **kwargs):
         super().__init__(**kwargs)
         self._regfile = regfile
         self._memory = memory
-        self._instruction_cache: Dict[int, Any] = {}
+        self._instruction_cache: Dict[int, Instruction] = {}
         self._icount = 0
         self._last_pc = None
+        self._last_executed_pc = None
```

Comment:
Seeing as this comes right after `_last_pc`, it would be worth adding a doc comment explaining the difference between the two, either here or on the property. Without understanding Manticore's internals it'd be easy to get mixed up.